src/index.html

```diff
@@ -13,20 +13,38 @@
     <title>Document</title>
     <link
       rel="stylesheet"
-      href="style.css"
+      href="./styles/style.css"
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <div class="search-bars">
+      <form
+        class="search-bar search-bar--big"
+        data-qa="big"
+      >
+        <div class="search-bar__icon search-bar__icon--big"></div>
```

Comment:
you don't need to use div to display the search icon, just use after pseudoclass for `<form>` for it. Please feel free to ask for help in fe_chat if something unclear to you.